iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -338,11 +338,11 @@ bool doesEndWithPathSeparator(const string<StringCapacity>& name) noexcept;
 /// @param[in] value of type F to convert to T
 /// @return converted value of F to corresponding value of T
 template <typename F, typename T>
-constexpr T from(const F value) noexcept;
```

Comment:
Why is it not possible to have a name in the declaration? There is a doxygen comment that refers to it. If it is a problem with some rule we need to disable the rule as documenting the parameters is required. Without a name order would be required to be unambiguous, which is arguably not ideal.

Please restore the name (if there is a rule problem we should fix it differently).